test/runnable/template1.d

```diff
@@ -775,21 +775,6 @@ void test30()
     assert(s == "Here it comes");
 }
 
-/******************************************/
-
-import std.string;
-
-template Foo31(alias X)
-{
-        alias X.toStringz y;
-}
-
-void test31()
-{
-    alias Foo31!(std.string) bar;
-}
```

Comment:
When was this introduced ? It looks like it is simply testing the ability to look up a symbol in an imported module, and doesn't need to import Phobos